examples/batch_solve.rs

```diff
@@ -184,44 +353,26 @@ fn main() -> Result<(), String> {
         target_exploitability
     );
 
-    // Save config to output directory
-
-    let config_json =
-        serialize_configs_to_json(&card_config, &tree_config, &added_lines, &removed_lines)?;
-
-    let config_contents = serde_json::to_string_pretty(&config_json).map_err(|e| e.to_string())?;
-    std::fs::write(&config_output_path, config_contents).map_err(|e| e.to_string())?;
-
-    let existing_board_files = boards
-        .iter()
-        .map(|b| dir.join(format!("{}.pfs", b.replace(" ", ""))))
-        .filter(|b| b.exists())
-        .collect::<Vec<PathBuf>>();
-
-    let boards_file_contents = boards.join("\n");
-    std::fs::write(&boards_output_path, &boards_file_contents).map_err(|e| e.to_string())?;
-
-    // Check if boards exist
-    if args.halt_on_existing && !existing_board_files.is_empty() {
-        println!("Halting. Board files already exist: ");
-        existing_board_files
-            .iter()
-            .for_each(|b| println!("- {}", b.display()));
-        exit(1);
-    }
-
+    // Begin the solving loop
```

Comment:
just a thought, but for testing it might be good to have a `--no-solve` option where we define the database but don't actually perform the solves. We should also have a way to solve unsolved files (this comes up when, for instance, there is an error during solves)

Reply:
But this might be an optional feature, not sure we need it immediately.

Reply:
Good point though, wouldn't be difficult to add.